qemu/tests/enable_scatter_windows.py

```diff
@@ -143,7 +147,7 @@ def _get_driver_version(session):
     vm.verify_alive()
     host_ip = utils_net.get_host_ip_address(params)
 
-    session = vm.wait_for_login(timeout=timeout)
+    session = vm.wait_for_serial_login(timeout=timeout)
```

Comment:
Hi all,
I found that the ws2016 of this patch failed consistently. After debugging, I modified this line 150 as well from ssh_login to serial_login cause it is the same as the patch [Netkvm: Fix session handling issue during netkvm driver installation #4158](https://github.com/autotest/tp-qemu/pull/4158/files#diff-e5666afb1caec7ffeb858d4d1cf15b6768ed711d7a1d6c858d2184d98c3f56eeR151).

It looks good for now. I will post the test result as soon as possible.

Thanks,
Wecom

Reply:
Cancelled this code change cause the problem has gone.